CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
  - Adding shortcut for loading Sprites and SpriteAnimation from the global cache
  - Adding loading methods for the different `ParallaxComponent` parts and refactor how the delta velocity works
  - Add tests for `Timer` and fix a bug where `progress` was not reported correctly
+ - move files to comply with the dart package layout convention
```

Comment:
Nit: I would use structure instead of layout here

Reply:
also, `Move`

Reply:
The convention is called layout: https://dart.dev/tools/pub/package-layout